app/models/user.rb

```diff
@@ -19,7 +24,13 @@ def social_worker?
     role == User::SOCIAL_WORKER
   end
 
-  validates :role, inclusion: { in: ROLES }
+  def department_manager?
+    role == User::DEPARTMENT_MANAGER
+  end
+
+  def superadmin_or_department_manager?
```

Comment:
Just my opinion, i like it more, when there are just the short_methods for one role and they get chained as in the body of this method, this keeps more extensible.